packages/react-core/src/components/TextArea/TextArea.tsx

```diff
@@ -36,6 +36,10 @@ export interface TextAreaProps extends Omit<HTMLProps<HTMLTextAreaElement>, 'onC
   value?: string | number;
   /** A callback for when the text area value changes. */
   onChange?: (event: React.ChangeEvent<HTMLTextAreaElement>, value: string) => void;
+  /** Callback function when text area is focused */
+  onFocus?: (event?: any) => void;
```

Comment:
can we be more specific about event types here?

Reply:
Should I also update the typing of TextInput and FormSelect? Those have `any` typing as well.

Reply:
technically changing they type for TextInput and FormSelect would be breaking.  So I would leave them.